--- include/netcdf.h.orig	2016-08-03 15:12:13.000000000 +0900
+++ include/netcdf.h	2016-08-03 15:17:39.000000000 +0900
@@ -262,9 +262,9 @@
 As a rule, NC_MAX_VAR_DIMS <= NC_MAX_DIMS.
 */
 /**@{*/
-#define NC_MAX_DIMS	1024
+#define NC_MAX_DIMS	65536
 #define NC_MAX_ATTRS	8192
-#define NC_MAX_VARS	8192
+#define NC_MAX_VARS	524288
 #define NC_MAX_NAME	256
 #define NC_MAX_VAR_DIMS	1024 /**< max per variable dimensions */
 /**@}*/
