--- src/utils/ncmpigen/Makefile.in.orig	2016-12-01 10:02:05.000000000 -0700
+++ src/utils/ncmpigen/Makefile.in	2016-12-01 10:05:23.000000000 -0700
@@ -51,18 +51,18 @@
 test:           $(PROGRAM) b-test c-test FORCE
 
 install: $(PROGRAM) $(MANUAL)
-	$(INSTALL) -d -m 755 $(MANDIR)/man1
-	$(INSTALL_DATA) $(srcdir)/$(MANUAL) $(MANDIR)/man1/$(MANUAL)
+	$(INSTALL) -d -m 755 $(DESTDIR)$(MANDIR)/man1
+	$(INSTALL_DATA) $(srcdir)/$(MANUAL) $(DESTDIR)$(MANDIR)/man1/$(MANUAL)
 
-	$(INSTALL) -d $(BINDIR)
-	$(INSTALL) -m 755 $(PROGRAM) $(BINDIR)/$(PROGRAM)
+	$(INSTALL) -d $(DESTDIR)$(BINDIR)
+	$(INSTALL) -m 755 $(PROGRAM) $(DESTDIR)$(BINDIR)/$(PROGRAM)
 
 uninstall:
-	$(RM) -f $(BINDIR)/$(PROGRAM)
-	$(RM) -f $(MANDIR)/man1/$(MANUAL)
+	$(RM) -f $(DESTDIR)$(BINDIR)/$(PROGRAM)
+	$(RM) -f $(DESTDIR)$(MANDIR)/man1/$(MANUAL)
 
 $(PROGRAM):	$(OBJS) $(LIBRARY)
-	$(LINK.c) $(OBJS) $(LDFLAGS) $(LIBS) 
+	$(LINK.c) $(OBJS) $(LDFLAGS) $(LIBS)
 
 # Below is used if a PnetCDF developer wants to rebuild ncmpigenyy.c or
 # ncmpigentab.c. In that case, configure.in at the root directory needs
